src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -49,7 +54,10 @@ describe('workspace service', () => {
         .expect(200);
       await Promise.all(
         listResult.body.result.workspaces.map((item: WorkspaceAttribute) =>
-          osdTestServer.request.delete(root, `/api/workspaces/${item.id}`).expect(200)
+          // this will delete reserved workspace
```

Comment:
I am a little bit confused for this comment, shall we add more details?

Reply:
sure, use workspace delete api will not able to delete reserved workspace, that will not able clean up the test data. Use saved objects api will not have this issue.

Reply:
Get it, could we add this info to this comment? It would be much easier to understand after adding the context you provide.

Reply:
i have drop the testing related commit, will have separate PR to add test cases.